recipes/planetaryimage/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "planetaryimage" %}
+{% set version = "0.5.0" %}
+{% set sha256 = "6b6af0d83147fe3fc4550ea753557010f01dd1bf905e060f7b942b3f16b38def" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy x.x
```

Comment:
Is it linking to `numpy`? For example, does it have Cython code or C/C++ code that makes use of NumPy? If not, `numpy` can be dropped from `build` and the `x.x` can be dropped from `run`.